dom-event-delegation/main.js

```diff
@@ -0,0 +1,18 @@
+// the querySelector method of the document object calls one argument a string assigns it to the constant variable '$clickButton'
+const $taskList = document.querySelector('.task-list');
+
+function handleClick(event) {
+  // will output the tagName with text content
+  //   OUTPUT EXAMPLE:
+  //   event.target : <span>Learn to Code</span['>' HERE BUT WILL CAUSE WHITESPACE ERROR]
+
+  console.log('event.target :', event.target);
+
+  // outputting to console the tagName (element name) of which
+  // the event.target fired.
+  // OUTPUT EXAMPLE:
+  // event.target.tagName : SPAN
+  console.log('event.target.tagName :', event.target.tagName);
+}
```

Comment:
You should have a lot more code inside of here. Make sure you're following the instructions. There should be a conditonal, and you should also be removing elements